docs/how-tos/integrating-with-webflow.md

```diff
@@ -0,0 +1,102 @@
+
+For the purposes of this tutorial, we will be very explicit on what things to test in your Webflow experiment. Keep in mind that all of the specific elements we’re testing are extendable to anything in your Webflow Environment.
+
+First, set up a Webflow landing page that has the following elements: a Header Title, a Header Subtitle, a Header CTA, and an image element. These elements should have no content in them for now since we will be filling in the image URL and messaging from our feature flag. As you are setting up these elements, make sure that you give each of the 4 elements one of the following custom ids: `header-subtitle`, `header-title`, `header-CTA`, and `header-image`. The custom code we add later specifically looks for these ids to write in the content from Eppo. See the screenshot below on where to update the ID for an element:
+
+![Setting up Webflow element IDs](/img/how-tos/integrating-with-webflow/setting-up-webflow-ids.png)
+
+Next, add the following code to your [custom code at the site level](https://university.webflow.com/lesson/custom-code-in-the-head-and-body-tags?topics=site-settings) in the footer:
+
+:::note
+
+By making sure our elements are “blank” to start with, we can completely reduce any flicker that would be caused by re-writing and element with Javascript. Make sure that you build in logic for default experience in the case that Eppo serves a `null` variation.
+
+:::
+
+```html
+<script>console.log('Loading eppo SDK')</script>
+<script src="https://cdn.jsdelivr.net/npm/@eppo/js-client-sdk@1.4.0/dist/eppo-sdk.js"></script>
+<script>console.log('Done loading eppo SDK')</script>
+
+<script>  
+// Init user parameters
+urlParams = new URLSearchParams(window.location.search);
+subject = urlParams.get('subject') || crypto.randomUUID();
+
+// Prep post-assignment code
+function setHeader() {
+  
+  // Get variation
+  const variationData = window.eppo.getInstance().getParsedJSONAssignment(subject, '<FEATURE-FLAG-KEY>');
+  console.log('>>> got variation data', variationData);
+  
+  // Add client side analytics tracking call to capture experiment data 
+  // and make sure assignment data is sent to your 
+  // data warehouse that is connected to Eppo.
+
+  document.getElementById('header-title').innerText = variationData.title;
+  document.getElementById('header-subtitle').innerText = variationData.subtitle;
+  document.getElementById('header-cta').querySelector('div').innerText = variationData.cta;
+  
+  const headerSrc = variationData.image;
+  
+  const headerImageDiv = document.getElementById('header-image');
+  headerImageDiv.style.backgroundImage = 'url('+headerSrc+')';
+  headerImageDiv.style.backgroundPosition = 'center center';
+  headerImageDiv.style.backgroundRepeat = 'no-repeat';
+  headerImageDiv.style.backgroundSize = 'auto 100%';
+}
+
+// Init Eppo Client
+opts = {apiKey: '<EPPO-SDK-KEY>'};
+window.eppo.init(opts).then(setHeader);
+window.eppo.getInstance().setIsGracefulFailureMode(false);
```

Comment:
You can probably strike this line. This will cause the SDK to throw an error instead of returning null if anything weird happens. Useful for debugging, but not something you want on prod.